tests/integration/telemetry/envoyfilter/customizemetrics/customize_metrics_test.go

```diff
@@ -60,6 +60,46 @@ const (
 	registryPasswd = "passwd"
 )
 
+func TestMetricDefinitions(t *testing.T) {
+	framework.NewTest(t).
+		Label(label.IPv4). // https://github.com/istio/istio/issues/35835
```

Comment:
do we need this? the issue is about making calls to external github.com

Reply:
will revisit this later